packages/babel-core/src/transformation/normalize-file.js

```diff
@@ -103,6 +104,10 @@ function parser(pluginPasses, options, code) {
           },
           options,
         );
+      const missingPluginMessage = generateMissingPluginMessage(missingPlugin);
```

Comment:
Just thinking aloud, but looking at this again... right now it outputs something like:

> This experimental syntax requires enabling the parser plugin: 'asyncGenerators' (1:14)
> \> 1 | async function* agf() {
>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^
>&nbsp;&nbsp;&nbsp;2 |   await 1;
>&nbsp;&nbsp;&nbsp;3 |   yield 2;
>&nbsp;&nbsp;&nbsp;4 | }
> Support for this experimental syntax isn't currently enabled. Add either @babel/plugin-syntax-async-generators (https://git.io/vb4SY) or @babel/plugin-syntax-async-generators (https://git.io/vb4SY) (if you only need parsing support) to the 'plugins' section of your Babel config.

Should we just omit the error from Babylon if `missingPlugin` and maybe just do:

> Support for the experimental syntax 'asyncGenerators' isn't currently enabled (1:14)
>
> \> 1 | async function* agf() {
>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^
>&nbsp;&nbsp;&nbsp;2 |   await 1;
>&nbsp;&nbsp;&nbsp;3 |   yield 2;
>&nbsp;&nbsp;&nbsp;4 | }
>
> Add either @babel/plugin-syntax-async-generators (https://git.io/vb4SY) or @babel/plugin-syntax-async-generators (https://git.io/vb4SY) (if you only need parsing support) to the 'plugins' section of your Babel config.

Reply:
Honestly, I thought about this. But I didn't know how it can be improved. The current message repeats the fact that there is some experimental plugin missing, twice.

I like the new message proposal. It's more meaningful and less repetitive. Can I go ahead and make this change?

Reply:
The original purpose for making the issue is that yes we would remove the original message and output the new one. I would only add the syntax part for things that we don't have a default transform for, otherwise it should only say the plugin. This is because the use case for babel is the transform not the syntax (because then you would be using babylon by itself)

Reply:
@hzoo @existentialism Then, the apt messages would be

>Support for the experimental syntax 'asyncGenerators' isn't currently enabled (1:14)
   \<codeFrame\>
Add @babel/plugin-proposal-async-generator-functions (https://git.io/vb4yp) to the 'plugins' section of your Babel config to enable transformation.

And, in the case where there is only a syntax plugin available

>Support for the experimental syntax 'dynamicImport' isn't currently enabled (1:14)
   \<codeFrame\>
Add @babel/plugin-syntax-dynamic-import (https://git.io/vb4Sv) to the 'plugins' section of your Babel config to enable parsing.

Reply:
Yea, I'm fine with that. Note, can we add line breaks around the codeframe:

> Support for the experimental syntax `[babylon plugin name]` isn't currently enabled ([loc]):
>
> [code frame]
>
> Add [npm package name] ([url]) to the 'plugins' section of your Babel config to enable [parsing|transformation].